src/pages/InputDemo/InputDemo.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { TextField, SelectField, RadioField } from '../../components';
+import { selectOptions, radioOptionsCricket, radioOptionsFootball } from '../../configs/constants'
+
+class InputDemo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      sport: '',
+      cricket: '',
+      football: '',
+    };
+  }
+
+  handleNameChange = (e) => {
+    this.setState({ name: e.target.value })
+    console.log(this.state);
+  };
+
+    handleSportChange = (e) => {
+      this.setState({ sport: e.target.value }, () => console.log(this.state));
+      if (e.target.value === 'Select') {
+        this.setState({ sport: '' });
```

Comment:
If target value is Select then all state related to select component will be blank string ex- sport: '', cricket: '', football: ''